ifrs17-template/Test/Data/ReportVariableBenchmarks/ReportVariableBenchmarks_2020_12_CH_MTUP10pct_Contractual.csv

```diff
@@ -0,0 +1,870 @@
+0,NIC,2020,USD,USD,C,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,-300,CF
+0,NIC,2020,USD,USD,C,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,299.67555,EOP
+0,NIC,2020,USD,USD,C,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,0.92261,IA
+0,NIC,2020,USD,USD,C,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,-299.67555,OCI1
+0,PR,2020,USD,USD,C,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,-798.53673,BOP
+0,PR,2020,USD,USD,C,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,400,CF
+0,PR,2020,USD,USD,C,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,-399.63393,EOP
+0,PR,2020,USD,USD,C,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,-1.0972,IA
+0,PR,2020,USD,USD,C,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,399.63393,OCI1
+0,NIC,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,598.75294,BOP
+0,NIC,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,-300,CF
+0,NIC,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,299.67555,EOP
+0,NIC,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,0.92261,IA
+0,NIC,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,-0.92261,IFIE1
+0,NIC,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,-298.75294,IR5
+0,NIC,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,299.67555,OCI1
+0,PR,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,-798.53673,BOP
+0,PR,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,400,CF
+0,PR,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,-399.63393,EOP
+0,PR,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,-1.0972,IA
+0,PR,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,1.0972,IFIE1
+0,PR,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,398.53673,IR5
+0,PR,2020,USD,USD,L,BE,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,-399.63393,OCI1
+0,PR,2020,USD,USD,L,BEPA,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,320,IR6
+0,,2020,USD,USD,,C,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,-82.02271,AM
+0,,2020,USD,USD,,C,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,139.90849,BOP
+0,,2020,USD,USD,,C,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,-15,EA
+0,,2020,USD,USD,,C,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,42.96811,EOP
+0,,2020,USD,USD,,C,CHF,DT5.1,P,False,LRC,ANN,N,Default,,DT5,0,CH,MTUP10pct,BBA,0.08233,IA
+0,,2020,USD,USD,,C,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,-0.08233,IFIE1
+0,,2020,USD,USD,,C,CHF,DT5.1,P,False,LRC,ANN,C,Default,,DT5,0,CH,MTUP10pct,BBA,82.02271,IR3
```

Comment:
Why do you have DT5.1 variables here? 
I think this is wrong. We should not show a report (same as BE) if no SCEN has been imported. 
Would you agree?
fyi @dtrzesniak

Reply:
It would be quite involved to change this behavior because the scenario can exist also in absence of transactional data for a given GOC, e.g. in presence of a scenario yield curve. Therefore, if we don't want to show a report same as BE for selected GOC, we need to check that we also don't have any parameter for that scenario. This I think would make the logic (in the report query) quite involved.

Reply:
Maybe we can look at this together and decide.
Intuitively, I would not show scenarios if they are exactly the same as BE. I can explain my reasons.